testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestAvroSchemaEvolution.java

```diff
@@ -13,166 +13,11 @@
  */
 package io.trino.tests.product.hive;
 
-import io.trino.tempto.AfterTestWithContext;
```

Comment:
damn. Only now I noticed that base class is based on preexisting class 🤦 .
Can you add a preparatory commit sequence which makes rename a rename, not delete and add.
1. rename `TestAvroSchemaEvolution.java` -> `BaseTestAvroSchemaEvolution.java` and add subclass `TestAvroSchemaEvolutionBasic.java`
2. (optionally) rename `TestAvroSchemaEvolutionBasic.java` -> `TestAvroSchemaEvolution.java`
3. Add `TestAvroSchemaEvolutionWithPartitioning.java` and make necessary changes to `BaseTestAvroSchemaEvolution.java`

Reply:
Oh, my mistake will rebase it accordingly.